ReactiveCocoa/UIKit/UITextView.swift

```diff
@@ -0,0 +1,38 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UITextView {
+	/// Sets the text of the text view.
+	public var text: BindingTarget<String> {
+		return makeBindingTarget { $0.text = $1 }
+	}
+
+	/// A signal of text values emitted by the text view upon end of editing.
+	public var textValues: Signal<String, NoError> {
+		var signal: Signal<String, NoError>!
```

Comment:
This would no longer be necessary after https://github.com/ReactiveCocoa/ReactiveSwift/pull/64 is landed.